### ISSUE TITLE
Windows, GUI freezes

### ISSUE THREAD

**Issue by argmantas** [xournalpp/xournalpp#2399]
After a couple of minutes of using xournal++ with my wacom I move the cursor to the toolbar of the programm and can't select anything. I can continue drawing but everything else is frozen. I downloaded the lastest version via the exe file. I am also running the latest windows update

**Comment by argmantas** [comment]
I noticed that the program doesn't crash when the wacom tablet is not connected. And a friend of mine had the same issue with the wacom intuos s

**Comment by Technius** [comment]
Can you take a screenshot of the About dialog? You can open it with `Help > About` in the menu bar at the top.

**Comment by argmantas** [comment]
![image](https://user-images.githubusercontent.com/74463199/99180206-978f7800-2724-11eb-96a5-76a59c259dc9.png)

**Comment by argmantas** [comment]
![Untitled](https://user-images.githubusercontent.com/74463199/99180246-da515000-2724-11eb-8e52-e3e5b8d67610.png)

**Comment by argmantas** [comment]
I also just installed it on arch and it works perfectly with the same tablet. some more information: the crash doesn't happen instantaneously It happens after about 5 to 10 minutes. And I'll try to describe the situation better. I am taking notes normally and after a certain amount of time when i move the cursor with the mouse to the toolbar i can't select anything. I can't even minimize the window or close the program. The only solution at this point is to kill the process from task manager.

**Comment by Febbe** [comment]
Can you also try it with the newest nightly?

**Comment by gherreip** [comment]
Hi there. 
Just discovered Xjournal++, it seems wonderful (to teach maths from home). But ... won't be able to use it.

I have exactly same problem with my XP Pen tablet. Suddenly all is frozen and I can only kill process from task manager.

**Comment by argmantas** [comment]
The same thing happens with the nightly

**Comment by LittleHuba** [comment]
Sounds like an issue with GTK. Could you try other applications using GTK and check if the same issue appears there? I assume Gimp would be suitable for this test. Please do use the tablet and touchscreen for input as the issue might be specific to your input devices.

**Comment by Febbe** [comment]
Can you also try to simultaneously use your fingers/mouse/stylus? This could trigger the bug.

**Comment by argmantas** [comment]
I dont't know why but after the latest windows update I tried to cause another crash by using all inputs but the program was stable

**Comment by gherreip** [comment]
I'm running Xournal on Windows 7. I can't hope any windows update anymore to help ....

**Comment by Leon358522** [comment]
Don't know if it's the same bug, but when my windows frezzes like yours (can't click with mouse on panels). I'm still able to navigate  through the panels with my pencil (also have wacom tablet). Running same version as argmantas

**Comment by gherreip** [comment]
I just tried to check that point, and it seems that it's the same for me : mouse disabled, pencil ok.

But I can only write "it seems", I'm not sure, because in the middle of my test, Xournal++ suddenly crashed. I was precisely (with the pencil) sliding a bmp file onto the page I was working on, when it completely stopped (pop up windows explaining me that *bin.exe stopped working). This sliding worked perfectly many times before that crash.

(Meanwhile, I can use Write from Stylus Lab without any kind of problems)

**Comment by dgillcrist** [comment]
I too use the Xoural++ with a huion writing pad, and have found that I find this same crash happens to me. If it happens again to me I can try to record my screen to demonstrate what happens.

**Comment by sajanpphilip** [comment]
I am facing the same issue in Windows 8. The xournalpp is frozen after using it for some time. I am using the latest version.

**Comment by hara1199** [comment]
I am facing the same problem as well. Wacom Intuos s tablet with windows 10 and the program freezes after two minutes of use. is there any solution?

**Comment by tharroun** [comment]
Same here with Wacom Intuos M and Win10. It's quite random, not related to any particular time or function use, so difficult to reproduce.

**Comment by fakemaxwell** [comment]
Getting the same issue here.  I think it has something to do with tablets and their drivers.  The new input system configured correctly seems to freeze less, as well as working in a saved file (rather than a new blank page).

Can reproduce by closing the Huion driver, opening xournal, and then opening the Huion driver.

**Comment by fakemaxwell** [comment]
Something odd- on my machine the GUI only freezes when using the mouse.  When I use the tablet to hit buttons, it works fine. This is with a Huion tablet, Windows 10.

**Comment by Sinthoras7** [comment]
I know, this issue is 2,5 years old, but since the update to Fedora 39, I now have the same problem. When I use the mouse everything is fine and when using the Tablet, the problems arise. 

> Sounds like an issue with GTK. Could you try other applications using GTK and check if the same issue appears there? I assume Gimp would be suitable for this test. Please do use the tablet and touchscreen for input as the issue might be specific to your input devices.

I tried using Gimp and there is the same issue, so this supports your suspicion.

**Comment by rolandlo** [comment]
@Sinthoras7 Is that under X11? Does it work fine on Wayland? Maybe you are suffering from #5249.

**Comment by Sinthoras7** [comment]
Yes, it was under X11. I tried Wayland right now and in Wayland everything is fine. Thank you for the indication!
